### PR TITLE
Update jquery.scrollTo.js

### PR DESCRIPTION
**Pull request by MathiasReker**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | update jquery.scrollTo.js to v2.1.2. When a customer clicks "Details" on an order in their history, it does not scroll to the bottom when Chrome is used. Chrome broke this around v61. Updating jquery.scrollTo.js to the current version (2.1.2) fixes this bug.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | this is fix for 1.7, 1.6 fix is here: https://github.com/PrestaShop/PrestaShop/pull/10644
| How to test?  | Using Chrome, click "Details" from order history. It should scroll to the bottom of the page where order the order details are shown

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/10645)

### CODE REVIEW

**Comment by MathiasReker** [comment]
Can this have 1.7.5.0 milestone?

**Comment by Quetzacoalt91** [comment]
Unfortunately, this PR can't be merged for 1.7.5.x, it should go to the develop branch.

**Comment by MathiasReker** [comment]
Okay. I changed the branch to develop.

File: js/jquery/plugins/jquery.scrollTo.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2007-2014 Ariel Flesler - aflesler<a>gmail<d>com | http://flesler.blogspot.com
+ * Copyright (c) 2007 Ariel Flesler - aflesler ○ gmail • com | https://github.com/flesler
```
Comment (mickaelandrieu) [inline js/jquery/plugins/jquery.scrollTo.js]: I'm worried about the rendering of the characters... are you using Windows?

Comment (MathiasReker) [reply js/jquery/plugins/jquery.scrollTo.js]: Yes, I am on Windows. I will look in to that.

Comment (MathiasReker) [reply js/jquery/plugins/jquery.scrollTo.js]: Solved, nice catch.

**Comment by MathiasReker** [comment]
All should be good now.

**Pull request closed without merging**